src/momento/vector_index_client.py

```diff
@@ -194,19 +194,19 @@ def upsert_item_batch(self, index_name: str, items: list[Item]) -> UpsertItemBat
         """
         return self._data_client.upsert_item_batch(index_name, items)
 
-    def delete_item_batch(self, index_name: str, ids: list[str]) -> DeleteItemBatchResponse:
+    def delete_item_batch(self, index_name: str, filter: list[str]) -> DeleteItemBatchResponse:
         """Deletes a batch of items from a vector index.
 
         Deletes any and all items with the given IDs from the index.
 
         Args:
             index_name (str): Name of the index to delete the items from.
-            ids (list[str]): The IDs of the items to be deleted from the index.
+            filter (list[str]): The IDs of the items to be deleted from the index.
```

Comment:
should we rename these arg descriptions to 

```
filter (list[str]): The filters used to delete items from the index
```

Reply:
In the future state this will be a filter expression. We will keep accepting a list of id's because that is a common case so we want to make it easy without wrapping them in a filter object.